server/org.eclipse.emfcloud.ecore.modelserver/src/main/org/eclipse/emfcloud/ecore/modelserver/EnotationPackageConfiguration.java

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2020 EclipseSource and others.
+ * Copyright (c) 2021 EclipseSource and others.
```

Comment:
Should be 2020-2021

Reply:
Actually, this file was newly created, it just appears as rename, as the `EcorePackageConfiguration` was removed.